src/redis.h

```diff
@@ -927,8 +928,12 @@ struct redisCommand {
     int lastkey;  /* The last argument that's a key */
     int keystep;  /* The step between first and last key */
     long long microseconds, calls;
+    sampleReservoire samples;
 };
 
+#define CMDTIME_PERCENTILES_COUNT 5
+const static double CMDTIME_PERCENTILES[] = {.25, .50, .90, .99, .999};
```

Comment:
This will create a different array for each translation unit.

This should be 
```
extern const double CMDTIME_PERCENTILES[CMDTIME_PERCENTILES_COUNT];
```
and then
```
const double CMDTIME_PERCENTILES[CMDTIME_PERCENTILES_COUNT];
```
in any translation unit (e.g., `redis.c`)